### PR TITLE
{bio}[goolf/1.7.20] cutadapt v1.16

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@pescobar Did this work for you? Fails for me with:

```
ERROR: Your Cython is at version '0.23.4', but at least version 0.24 is required
```

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2034.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ed713d726fd57950885cd282f7ba3e5e for a full test report.

File: easybuild/easyconfigs/c/cutadapt/cutadapt-1.16-goolf-1.7.20-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
```
Comment (boegel) [inline easybuild/easyconfigs/c/cutadapt/cutadapt-1.16-goolf-1.7.20-Python-2.7.11.eb]: should be `easybuilders`, not `hpcugent` (anymore), fixed in https://github.com/pescobar/easybuild-easyconfigs/pull/66

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): uses a deprecated toolchain.
For more information on deprecated toolchains, see https://easybuild.readthedocs.io/en/latest/Deprecated-easyconfigs.html#deprecated-toolchains.
Please consider reopening or submitting a new PR using a supported toolchain.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**